packages/server/src/fhir/operations/patientsummary.ts

```diff
@@ -267,9 +281,6 @@ export class PatientSummaryBuilder {
       case 'Observation':
         this.chooseSectionForObservation(resource);
         break;
-      case 'Task':
```

Comment:
My original plan was to use `Task` resources for a lot of the "Plan of Treatment" data.  I changed course on that, and settled on `CarePlan` + `ServiceRequest`, which appears to be more aligned with the IPS Patient Summary recommendations.